vk_helper.py

```diff
@@ -579,6 +579,7 @@ def generateSizeHelper(self):
         self.size_helper_gen.setCopyright(self._generateCopyright())
         self.size_helper_gen.setHeader(self._generateSizeHelperHeader())
         self.size_helper_gen.setBody(self._generateSizeHelperFunctions())
+        self.size_helper_gen.setFooter(self._generateSizeHelperFooter())
```

Comment:
As this file resides in LVL repo, better to make this change in LVL instead of VT.